modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldDefaultValue.testcase

```diff
@@ -82,7 +169,7 @@ definition {
 				fieldName = "customField",
 				fieldType = "String",
 				isRequired = "false",
-				objectName = "CustomObject181204");	
+				objectName = "CustomObject181204");
```

Comment:
Please fix SF errors in the commit that caused them. We shouldn't be fixing them in other commits.